app/assets/stylesheets/components/_widget.scss

```diff
@@ -0,0 +1,28 @@
+.widget {
+  background: white;
+  box-shadow: 0px 0px 1px lighten($black, 50%);
+  margin: 0.8rem;
+
+  @media #{$small-only} { margin: 0.5rem; }
+
+  .main, .header { padding: 1.4rem; }
+
+  .header {
+    padding-bottom: 0;
+
+    h1, h2, h3 {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own line